src/wp-includes/functions.php

```diff
@@ -9037,3 +9064,22 @@ function wp_admin_notice( $message, $args = array() ) {
 
 	echo wp_kses_post( wp_get_admin_notice( $message, $args ) );
 }
+
+/**
+ * Checks if a mime type is for a HEIC/HEIF image.
+ *
+ * @since 6.7.0
+ *
+ * @param string $mime_type The mime type to check.
+ * return boot Whether the mime type is for a HEIC/HEIF image.
```

Comment:
```suggestion
 * return bool Whether the mime type is for a HEIC/HEIF image.
```

Typo: leaving it to Adam, et al, to review the code changes.

Reply:
```suggestion
 * @return bool Whether the mime type is for a HEIC/HEIF image.
```

Besides the already caught bool typo an `@` was missing in front of the

Reply:
Hm, cannot add that typo fix from here :(